code/modules/clothing/under/miscellaneous.dm

```diff
@@ -218,6 +218,23 @@
 	worn_state = "nt_skirtsuit"
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
 
+/obj/item/clothing/under/suit_jacket/skirtsuit/color
+	desc = "A black coat with an colour coordinated kerchief accompanied by a swept skirt with a tasteful stripe. Works for every occasion."
+	icon_state = "skirtsuit_recolourable"
+	item_state = "skirtsuit_recolourable"
+	worn_state = "skirtsuit_recolourable"
+	build_from_parts = TRUE
+	worn_overlay = "base"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+
+/obj/item/clothing/under/suit_jacket/skirtsuit_command
+	desc = "A black coat with a gold kerchief accompanied by a swept skirt with a respectable golden trim. Works only for Women in Power."
```

Comment:
```suggestion
	desc = "A black coat with a gold kerchief accompanied by a swept skirt with a respectable golden trim. Works for every occasion."
```

Reply:
This was a tongue in Cheek joke intentionally put here to imply that if you are wearing this you better be a Head of Staff or you will have to answer for yourself
I hope this tongue in cheek humor can stay, if not then I will change it.

Reply:
`Works only for people in power."` would work too, of course.